pipelines/optimus/README.md

```diff
@@ -7,3 +7,36 @@ The Optimus pipeline is a pipeline for processing 3' single-cell expression data
 Optimus is a alignment and transcriptome quantification pipeline. Optimus corrects Cell Barcodes (CBs) and Unique Molecular Identifiers (UMIs), aligns reads to the genome, generates an expression count matrix in a UMI-aware manner, detects empty droplets, calculates summary statistics for genes and cells, and returns outputs in BAM and Zarr file formats. Special care is taken to keep  all reads that may be useful to the downstream user, such as unaligned reads or reads with uncorrectable barcodes. This design provides flexibility to the downstream user and allows for alternative filtering or leveraging the data for novel methodological development.
 
 More details can be found in the [Optimus Acceptance Report](https://docs.google.com/document/d/158ba_xQM9AYyu8VcLWsIvSoEYps6PQhgddTr9H0BFmY/edit)
+
+
+### Installation  
+The Optimus pipeline code can be downloaded by cloning the github repository [Skylab](https://github.com/HumanCellAtlas/skylab/) 
+
+### Requirements  
+Optimus can deployed using [Cromwell](https://software.broadinstitute.org/wdl/), a GA4GH compliant, flexible workflow management system that supports multiple computing platforms
+
+### Running Optimus
+* [Optimus.wdl] (https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/Optimus.wdl) in the pipelines/optimus folder,  of the repository, implements the workflow by importing individual tasks in task based WDLs in skylab/library.
+
+#### Inputs
+The input file locations are specified in a json file, e.g., [here](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/example_test_inputs.json).
+
+#### Outputs
+The outputs from the Optimus pipeline can be identified from the outputs of the individual tasks, e.g. [here](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/example_test_outputs.json)
+
+Following are the the types of files produced from the pipeline.
+
+| file type | Description |
+| --- | --- |
+| pipeline_version | Version of the pipeline |
+| bam | Merged and Sorted BAM file |
+| matrix |  Sparse count matrix in numpy format |
+| matrix_row_index | Sparse count matrix row names in numpy format |
+| matrix_col_index | Sparse count matrix col names in numpy format |
+| cell_metrics | Cell metrics table in text format |
+| gene_metrics | Gene metrics table in text format |
+| cell_calls | Cell metadata from emptyDrops |
+| zarr_outputs_file | Count matrix and cell and gene metrics in zarr format|
+
+### Contributing to Optimus
```

Comment:
This reads more like "where are the parts of Optimus" than "How can you contribute improvements". If we want contribution I think this should link out to contributing.md files in the various repositories, or describe how a user should contribute a modification to Optimus. If we don't know, it could be as simple as "start with an issue", or giving them the pipelines-team email and inviting them to get in touch. 

If we're not ready for contribution, I'd rename this to "Components of Optimus".

Reply:
Yes, it makes sense to change it to "Components of Optimus" for now.